ci/builders/standalone/linux_android_emulator_skia.json

```diff
@@ -0,0 +1,81 @@
+{
+    "gclient_variables": {
+        "use_rbe": true
+    },
+    "gn": [
+        "--android",
+        "--android-cpu=x64",
+        "--no-lto",
+        "--rbe",
+        "--no-goma",
+        "--target-dir",
+        "ci/android_emulator_skia_debug_x64"
+    ],
+    "dependencies": [
+        {
+            "dependency": "goldctl",
+            "version": "git_revision:720a542f6fe4f92922c3b8f0fdcc4d2ac6bb83cd"
+        }
+    ],
+    "name": "ci/android_emulator_skia_debug_x64",
+    "description": "Build for debug mode x64 Android Skia scenario app tests.",
+    "ninja": {
+        "config": "ci/android_emulator_skia_debug_x64",
+        "targets": [
+            "flutter/impeller/toolkit/android:unittests",
+            "flutter/shell/platform/android:flutter_shell_native_unittests",
+            "flutter/testing/scenario_app"
+        ]
+    },
+    "tests": [
+        {
+            "language": "dart",
+            "name": "Android Scenario App Integration Tests (Skia)",
+            "test_timeout_secs": 900,
+            "max_attempts": 2,
+            "test_dependencies": [
+                {
+                    "dependency": "android_virtual_device",
+                    "version": "android_35_google_apis_x64.textpb"
```

Comment:
Should this file encode the "35" nature in some way?  "ci/android_emulator_skia_debug_x64" is duplicate, but "ci/android_emulator_skia_debug_x64/API35" is more descriptive

It took me having to copy these two files and diff them locally to spot the difference

Reply:
I completely blindly copied one file to the other, the only difference is that I removed two levels of nesting:
https://github.com/flutter/engine/blob/38e37ce9879b2efe4890a8d09f4207dc5686a1bd/ci/builders/linux_android_emulator_skia_34.json#L1-L2
Like this:
https://github.com/flutter/engine/pull/55186/commits/595cdc2ce336b3677da29c8c8837bd2da38b3781?diff=split&w=1

Can you file an issue if you'd like it changed? Someone from the Android team should think that through for future bumps https://github.com/flutter/engine/pull/54186/files#diff-ee89d2cc6e90e32301d49469c8a095fc1f92a33526471b2f71f13fea468fa225L77